src/main/java/fr/rader/boblite/Main.java

```diff
@@ -23,242 +29,123 @@ public void start() {
-            case "1.9.1":
-            case "1.9.2":
-            case "1.9.3":
-            case "1.9.4":
-            case "1.10":
-            case "1.10.1":
-            case "1.10.2":
-            case "1.11":
-            case "1.11.1":
-            case "1.11.2":
-                timePacketID = 0x44;
-                weatherPacketID = 0x1E;
-                chatPacketID = 0x0F;
-                break;
-
-            case "1.12":
-            case "1.12.1":
-            case "1.12.2":
-                timePacketID = 0x47;
-                weatherPacketID = 0x1E;
-                chatPacketID = 0x0F;
-                break;
-
-            case "1.14":
-            case "1.14.1":
-            case "1.14.2":
-            case "1.14.3":
-            case "1.14.4":
-                timePacketID = 0x4E;
-                weatherPacketID = 0x1E;
-                chatPacketID = 0x0E;
-                break;
-
-            case "1.15":
-            case "1.15.1":
-            case "1.15.2":
-                timePacketID = 0x4F;
-                weatherPacketID = 0x1F;
-                chatPacketID = 0x0F;
-                break;
-
-            case "1.16":
-            case "1.16.1":
-            case "1.16.2":
-            case "1.16.3":
-            case "1.16.4":
-            case "1.16.5":
-                timePacketID = 0x4E;
-                weatherPacketID = 0x1D;
-                chatPacketID = 0x0E;
-                break;
-
-            case "1.17":
-            case "1.17.1":
-                timePacketID = 0x58;
-                weatherPacketID = 0x1E;
-                chatPacketID = 0x0F;
-                break;
+        if (projectFiles == null) {
+            System.out.println("'" + projectFolder.getAbsolutePath() + "' is not a valid path.");
+            System.exit(0);
+        }
 
-            case "1.18":
-            case "1.18.1":
-                timePacketID = 0x59;
-                weatherPacketID = 0x1E;
-                chatPacketID = 0x0F;
-                break;
+        // Load .mcpr files from the project folder.
+        List<ReplayData> replays = new ArrayList<ReplayData>();
+        for (File file : projectFiles) {
+            if (file.getName().endsWith(".mcpr") && !file.isDirectory()) {
+                replays.add(new ReplayData(
+                        file,
+                        projectFolder,
+                        this));
+            }
+        }
 
-            // we show an error and stop if the protocol isn't supported
-            default:
-                JOptionPane.showMessageDialog(null, "Error: unsupported Minecraft version: " + replayData.getMetaData("mcversion"));
-                return;
+        // If no .mcpr files existed in the project folder, ask the user to select one or more .mcpr files.
+        if(replays.isEmpty()){
+            File[] files = IO.openFilePrompt(OS.getMinecraftFolder() + "replay_recordings/", "Replay File", "mcpr");
+            if(files != null) {
```

Comment:
Add a space after the `if`

Reply:
Resolved in commit [fix code formatting - 63d35e3](https://github.com/RaderRMT/BobLite/pull/1/commits/63d35e385e87abea775c880b09dfc2494ea08a84)